lib/draper/automatic_delegation.rb

```diff
@@ -6,8 +6,12 @@ module AutomaticDelegation
     def method_missing(method, *args, &block)
       return super unless delegatable?(method)
 
-      self.class.delegate method
-      send(method, *args, &block)
+      if self.class.method_defined?(method)
+        object.send(method, *args, &block)
```

Comment:
Another possible option is to check if method defined and fail with explicit error message with instructions how to fix issue.